src/sparsetensors/quantization/quant_config.py

```diff
@@ -90,3 +95,41 @@ class QuantizationConfig(BaseModel):
     quantization_status: QuantizationStatus = QuantizationStatus.INITIALIZED
     global_compression_ratio: Optional[float] = None
     ignore: Optional[List[str]] = None
+
+    @staticmethod
+    def from_pretrained(model: Module) -> "QuantizationConfig":
+        """
+        TODO: fill in docstrings
+        """
+        quant_scheme_to_layers = []
+        quantization_status = None
+        ignore = []
+        for name, submodule in iter_named_leaf_modules(model):
```

Comment:
See TODO comment about allowing for exceptions in leaf nodes for observers. This will be relevant for non frozen quantized models